umbraco/Umbraco-CMS
# V9: Update MaxRequestBodySize on both IIS Server and Kestrel Server

## Details
This PR is made as a follow-up fix based on the discussion in #10930.
It makes sure that we update the MaxRequestBodySize based on what value we have configured for `Umbraco:CMS:Runtime:MaxRequestLength` (_[docs](https://our.umbraco.com/documentation/Reference/V9-Config/RuntimeSettings/)_) - If no value is specified, the `MaxRequestBodySize` value on the server will be set to null or in other words to unlimited.

It fixes the back-end for the reported #10920 issue. The front-end should be investigated further in order to completely address the reported problem. 

## Test
- Run Umbraco with IIS Express (IIS Server) or Umbraco.Web.UI (Kestrel Server);
- Configure Runtime settings as shown on the docs example;
- Make sure that the value set in `MaxRequestLength` updates `IISServerOptions.MaxRequestBodySize` or `KestrelServerOptions.MaxRequestBodySize` depending on how are you running Umbraco - set a break point in **ConfigureIISServerOptions.cs** and **ConfigureKestrelServerOptions.cs** and check the values (you no longer should see the default 30,000,000 bytes).


-----
The problem with the front-end is that when a file is bigger than the specified limit, no "_Max file size_" error is shown on the UI.
